packages/components/src/DateInput/DateInputDescription/DateInputDescription.tsx

```diff
@@ -19,7 +19,7 @@ export const DateInputDescription = ({
     React.isValidElement(description)
 
   const translatedInputFormatMessage = formatMessage({
-    id: "filterDateRangePicker.inputFormat",
+    id: "dateInputDescription.inputFormat",
```

Comment:
Renamed because moved out of FilterDRP. Also renamed within all the locale files.